odoo/tests/common.py

```diff
@@ -622,6 +637,8 @@ def _websocket_send(self, method, params=None):
         """
         send chrome devtools protocol commands through websocket
         """
+        if self.ws is None:
```

Comment:
perhaps this fix belongs in a separate commit, to keep things clear?

Reply:
yep